paymentsheet/src/main/java/com/stripe/android/paymentsheet/IntentConfirmationHandler.kt

```diff
@@ -279,6 +380,27 @@ internal class IntentConfirmationHandler(
         ) : Result
     }
 
+    /**
+     * Action to perform if a user cancels a running confirmation process.
+     */
+    enum class CancellationAction {
+        /**
+         * This actions means the user has cancels a critical confirmation step and that the user should be notified
+         * of the cancellation if relevant.
+         */
+        InformCancellation,
+
+        /**
+         * This action means that the user has asked to modify the payment details of their selected payment option.
+         */
+        ModifyPaymentDetails,
+
+        /**
+         * Means no action should be taken if the user cancels a step in the confirmation process.
+         */
+        None,
+    }
+
```

Comment:
Don't really like the naming for these but couldn't think of anything better for describing them.